smart-contracts/Makefile

```diff
@@ -0,0 +1,24 @@
+# Makefile
+
+SHELL:=/bin/bash
+
+setup:
+	python3 -m venv venv
+	source ./venv/bin/activate
+	pip3 install -r requirements.txt
+	yarn
+
+# Run slither over entire directory
+# `make slither`
+slither: setup
+	slither . || true
```

Comment:
Sorry, I've been spending way too much time in bash scripts when I suggested `|| true` - make has it's own (better) way of doing this.  Just add - at the beginning of the line:

`<tab>-slither .`

and an error won't cause make to stop.